code/datums/supplypacks/misc.dm

```diff
@@ -322,3 +320,66 @@
 	cost = 50
 	containertype = /obj/structure/closet/crate
 	containername = "Neon Lights crate"
+
+//Culture Crates
+/datum/supply_pack/misc/culture_apidean
+	name = "Apidean Culture Crate"
+	contains = list(
+			/obj/fiftyspawner/wax = 2,
+			/obj/item/healthanalyzer/apidean = 1,
+			/obj/item/analyzer/apidean = 1,
+			/obj/item/reagent_scanner/apidean = 1
+			)
+	cost = 50
+	containertype = /obj/structure/closet/crate
+	containername = "Apidean Culture crate"
+
+/datum/supply_pack/misc/weaponry_apidean
```

Comment:
I don't really feel like weapon crates should be under Misc, even if they're security locked- consider moving to Munitions, which is where most dedicated weapons and ammo go?

Reply:
For that matter, the Tyrmalin mining crate should probably go under Supplies too, to be next to the Belt Miner equipment.